tests/lib/tools/suite/journal-state/task.yaml

```diff
@@ -0,0 +1,37 @@
+summary: smoke test for the journal-state tool
+
+execute: |
+    # Run shellcheck for journal-state script
+    if [ "$SPREAD_SYSTEM" = "ubuntu-18.04-64" ]; then
+        shellcheck "$TESTSTOOLS"/journal-state
```

Comment:
as mentioned above I think this test should install shellcheck manually here

Reply:
see previous answer